spec/controllers/projects_controller_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe ProjectsController, :type => :controller do
   render_views
 
-  describe 'GET /' do
+  describe 'GET /projects' do
```

Comment:
We should probably just get rid of this index route, you think? We won't actually be using it so I don't think people should be able to navigate to it.